packages/components/src/ui/ariakit-tooltip/test/index.tsx

```diff
@@ -0,0 +1,330 @@
+/**
+ * External dependencies
+ */
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+/**
+ * WordPress dependencies
+ */
+import { shortcutAriaLabel } from '@wordpress/keycodes';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../../../button';
+import { ToolTip } from '..';
+import { TOOLTIP_DELAY } from '../../../tooltip/';
+
+const props = {
+	children: <Button>Button</Button>,
+	text: 'tooltip text',
+	delay: TOOLTIP_DELAY,
+};
+
+describe( 'ToolTip', () => {
+	it( 'should not render the tooltip if multiple children are passed', async () => {
+		const user = userEvent.setup();
+
+		render(
+			// @ts-expect-error
```

Comment:
It's good practice to add a little inline comment explaining why we're expecting a TypeScript error